server/src/main/resources/db/migration/V1_167__add_more_standard_tasks.sql

```diff
@@ -53,7 +53,7 @@ values ('Residence Attestation','UploadTask',
 insert into task (name, task_type, description, days_to_complete, upload_subfolder_name,
                   upload_type, uploadable_file_types,
                   created_by, created_date)
-values ('Residence Attestation','UploadTask', 'Translation of original document',
+values ('Residence Attestation Translation','UploadTask', 'Translation of original document',
```

Comment:
Migration `V1_184__task_display_name.sql` fails because it's adding a unique constraint on the `name` column but this task has the same name as the one above.

Reply:
Good pick up - many thanks